dotcom-rendering/src/components/Masthead/Titlepiece/EditionDropdown.tsx

```diff
@@ -89,15 +88,13 @@ export const EditionDropdown = ({
 
 	return (
 		<div css={editionDropdownStyles}>
-			<Island priority="critical">
-				<Dropdown
-					label={label}
-					links={linksToDisplay}
-					id="masthead-edition"
-					dataLinkName={dataLinkName}
-					cssOverrides={dropDownOverrides}
-				/>
-			</Island>
+			<Dropdown
```

Comment:
Since the whole titlepiece is hydrated, this doesn't need to be set as an island anymore.